src/main/java/space/testflight/FlywayExtension.java

```diff
@@ -127,6 +127,8 @@ public void afterAll(ExtensionContext context) throws Exception {
     if (getDatabaseInstance(context) == Flyway.DatabaseInstanceScope.PER_TEST_CLASS) {
```

Comment:
Is this if statement required any more?

Reply:
Since I added a line for stopping the container in the global store, it might be necessary to close the container in the class store, if that's different. I think currently it's not, but it won't hurt for now (see response to your other remark).